.github/workflows/build-test-deploy.yaml

```diff
@@ -551,6 +551,17 @@ jobs:
           replacesArtifacts: true
           token: ${{ secrets.GITHUB_TOKEN }}
           artifacts: "qTox-nightly.dmg,qTox-nightly.dmg.sha256"
+  build-osx-users:
+    name: macOS user
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
There is a v3 of this now.

Reply:
Ah good to know, will update all of them in a new PR.

Reply:
Update PR is here: https://github.com/qTox/qTox/pull/6600